docs/_includes/common/progress-bar.html

```diff
@@ -0,0 +1,21 @@
+<!-- Auto generated code -->
+<div id="progress-bar">
+    {% include common/section-header.html name="progress-bar" version=page.versions.progress-bar %}
+
+    <h3>The Progress Bar component allows the user to show progress on a task.</h3>
+
+    <div class="demo">
+        <div class="demo__inner">
+            <progress style="margin: 25px;" class="progress" value=40 min=0 max=100>40%</progress>
```

Comment:
Margin should be applied directly to the module. I'd say `margin: 8px 0` for small screen. 16px 0 for large.